packages/netlify-cms-widget-markdown/src/MarkdownControl/index.js

```diff
@@ -34,9 +36,12 @@ export default class MarkdownControl extends React.Component {
   constructor(props) {
     super(props);
     editorControl = props.editorControl;
+    allowedModes = [...(props.field.get('mode') ?? ['raw', 'rich_text'])];
+    const preferredMode = localStorage.getItem(MODE_STORAGE_KEY) ?? 'rich_text';
+
     _getEditorComponents = props.getEditorComponents;
     this.state = {
-      mode: localStorage.getItem(MODE_STORAGE_KEY) || 'visual',
+      mode: allowedModes.indexOf(preferredMode) !== -1 ? preferredMode : allowedModes[0],
```

Comment:
Using `rich_text` instead of `visual` will actually break existing configuration for users that have `visual` in their local storage.

You can fix it by reversing the default `allowModes` to `['rich_text','raw']` so the fallback is `rich_text`

Reply:
👍 Done.